SettingsList.py

```diff
@@ -3739,20 +3739,29 @@ def __init__(self, name, gui_text, min, max, default, step=1,
         choices        = {
             'off': 'Off',
             'textures': 'Texture',
-            'sizes': 'Size and Texture'
+            'both':  'Both Size and Texture',
+            'classic': 'Classic'
         },
         gui_tooltip    = '''\
-            If enabled, chest texture will reflect its contents
+            
+			
+			If "Texture" is enabled, chest texture will reflect its contents
```

Comment:
Sorry for the nitpick, but could you change these tabs to spaces? Everything else seems good to me!

Reply:
Coulda sworn I got all those :(.   Hate how it adds tabs whenever you hit return lol, will fix tonight.

Reply:
You might need to look into a better editor that can figure out tabs vs. spaces based on the file/project lmao.

> Sorry for the nitpick, but could you change these tabs to spaces? Everything else seems good to me!

I'd say not even really a nitpick, especially in a language that utilizes significant whitespace. :)